ir/instr.cpp

```diff
@@ -4004,23 +4013,51 @@ StateValue X86IntrinBinOp::toSMT(State &s) const {
 
     return rty->aggregateVals(vals);
   }
+  case mmx_padd_b:
+  case mmx_padd_w:
+  case mmx_padd_d:
```

Comment:
as these adds are all vertical, can you merge these 3 cases with the pavg codes.